core/src/test/java/org/apache/iceberg/hadoop/TestHadoopCommits.java

```diff
@@ -421,7 +423,13 @@ public void testConcurrentFastAppends(@TempDir File dir) throws Exception {
         TABLES.create(
             SCHEMA,
             SPEC,
-            ImmutableMap.of(COMMIT_NUM_RETRIES, String.valueOf(threadsCount)),
+            ImmutableMap.of(
+                COMMIT_NUM_RETRIES,
+                String.valueOf(threadsCount),
+                COMMIT_MIN_RETRY_WAIT_MS,
+                "10",
```

Comment:
Without your changes the test takes 48s for me locally. With your changes it takes 37s. Adding COMMIT_MAX_RETRY_WAIT_MS = "1000" brings it down to 22s, so I would also specify this setting

Reply:
also what about setting LOCK_ACQUIRE_INTERVAL_MS = "100"?

Reply:
It's not used with `LOCK_ACQUIRE_TIMEOUT_MS=0`